proto/aserto/directory/assertion/v3/assertion.proto

```diff
@@ -0,0 +1,174 @@
+syntax = "proto3";
+
+package aserto.directory.assertion.v3;
+
+option go_package = "github.com/aserto-dev/go-directory/aserto/directory/assertion/v3;assertion";
+
+import "google/protobuf/empty.proto";
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+import "aserto/directory/common/v3/common.proto";
+import "aserto/directory/reader/v3/reader.proto";
+
+
+service Assertion {
+    rpc GetAssertion(GetAssertionRequest) returns (GetAssertionResponse) {
+        option (google.api.http) = {
+            get: "/api/v3/directory/assertion"
```

Comment:
Wouldn't we want the REST endpoint to be `/api/v3/directory/assertions/{id}`?

Reply:
Actually, to stay consistent with the reader/writer APIs which do use different paths for singular and plural operations, this should be `/api/v3/directory/assertion/{id}`.